java/client/build.gradle

```diff
@@ -189,6 +189,8 @@ tasks.register('copyNativeLib', Copy) {
         from "${projectDir}/../target/${arch}-unknown-linux-gnu/release/"
     } else if (osdetector.os == 'linux' && osdetector.release.id == 'alpine') {
         from "${projectDir}/../target/${arch}-unknown-linux-musl/release/"
+    } else {
+        from "${projectDir}/../target/release/"
```

Comment:
This part was causing a problem on OSX, the other change below is for Ubuntu (non-Alpine Linux distros).